solr/docker/include/scripts/solr-fg

```diff
@@ -15,21 +15,19 @@ if [[ -z "${OOM:-}" ]]; then
   OOM='none'
 fi
 case "$OOM" in
```

Comment:
Should this also be moved to bin/solr?

Reply:
I don't think so. It might make sense to remove it entirely, but in bin/solr these options can be specified directly instead.

Reply:
I wonder what effect these even have if we are going to run the oom_script by default, which will kill the solr process

Reply:
After some experimentation, I don't think these do anything.